docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,6 +6,18 @@
 
 :::
 
+## V1.9.5 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.6))
+
+This version is backwards compatible to
+[v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0). It is
+optional, but encouraged. The supported plugin version is `20`.**
+
+**Change 1**
```

Comment:
I added a *DO NOT MERGE* label until these change details are fleshed out.

Reply:
We might want to make these PR drafts in the future so we can still leave feedback on them, but block merging automatically